ipfx/_schemas.py

```diff
@@ -4,9 +4,11 @@
 
 class SweepFeatures(DefaultSchema):
     stimulus_code = String(description="stimulus code", required=True)
+    stimulus_name = String(description="stimulus name", required=True)
```

Comment:
You should be able to get this (`stimulus_name`) from the ontology instead of the schema, no?

Reply:
Removed stimulus_name from the schema